### PR TITLE
Velero in okteto

### PR DESCRIPTION
**Pull request by syntastical**
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: cmd/kots/cli/velero.go [hunk 1]
```diff
@@ -81,11 +81,14 @@ func VeleroEnsurePermissionsCmd() *cobra.Command {
 				return errors.Wrap(err, "failed to get clientset")
 			}
 
-			if err := snapshot.EnsureVeleroPermissions(cmd.Context(), clientset, veleroNamespace, kotsadmNamespace); err != nil {
-				return err
+			skipRbac := v.GetBool("skip-rbac")
```
Comment (sgalsaleh) [inline cmd/kots/cli/velero.go:84]: can we have this consistent with the other commands? which i believe is called `ensure-rbac`

Comment (syntastical) [reply cmd/kots/cli/velero.go:84]: Would they need then specificy `--ensure-rbac false` and the default would be true?

Comment (sgalsaleh) [reply cmd/kots/cli/velero.go:84]: `--ensure-rbac=false`, and yes.

Comment (syntastical) [reply cmd/kots/cli/velero.go:84]: Feels a little clunky, how about a compromise.  We use `skip-rbac-check` already for other commands, how do you feel about me changing `skip-rbac` to `skip-rbac-check`?

**Pull request closed without merging**